packages/block-editor/src/components/block-vertical-alignment-toolbar/index.js

```diff
@@ -68,10 +68,10 @@ export default compose(
 	} ),
 	withViewportMatch( { isLargeViewport: 'medium' } ),
 	withSelect( ( select, { clientId, isLargeViewport, isCollapsed } ) => {
-		const { getBlockRootClientId, getEditorSettings } = select( 'core/editor' );
+		const { getBlockRootClientId, getSettings } = select( 'core/block-editor' );
```

Comment:
This fix is unrelated but I thought I'd include here as a small fix (the block editor module should not rely on the editor data module) cc @getdave

Reply:
Thanks